sessions/smoother-collaboration.qmd

```diff
@@ -402,28 +465,31 @@ We've covered how to adhere to a style guide in both the introductory as
 well as the intermediate course, because it is such a useful and
 powerful tool to easily write more readable code. We'll expand on it in
 this course because it fits precisely with the theme of collaboration.
-That's because, when you're working on your own and not needing to worry
+That's because, when you're working on your own and don't need to worry
 about anyone seeing your code, there's a natural temptation to write
-your code like you might write notes to yourself... scribbled and
-scrawled down quickly. But when working with others, how it looks can
-greatly impact how quickly and easily others are able to read and
-interpret your code. Multiply all the collaborators on a project with
-this natural temptation for quick (and sloppy) coding, you can imagine
-how easy it is for code to massively "drift" towards being poorly
-formatted, especially when deadlines are close.
-
-That's when "linters" or "stylers" (types of "static code analysis"
-tools) become very useful. They will scan your code for common mistakes
-or syntax problems and either list them out for you to fix or fix them
-for you automatically. Linters are great when you are collaborating on a
-project with collaborators who are not as experienced in writing code or
-who only occasionally contribute so don't know the workflow culture of
-your project. In this way, you might want to have automatic
-linting/styling checks that are independent of you having to run them
-yourself. This is where the `{styler}` package comes in!
+your code like you might write notes to yourself: scribbled and scrawled
+down quickly.
+
+But when working with others, how it looks can greatly impact how
+quickly and easily others are able to read and interpret your code.
+Multiply all the collaborators on a project with this natural temptation
+for quick (and potentially sloppy) coding, you can imagine how easy it
+is for code to massively "drift" towards being poorly formatted,
+especially when deadlines are approaching.
+
+That's when "linters" or "stylers" (types of static code analysis tools)
+become very useful. They scan your code for common mistakes or syntax
+issues, either listing them for you to fix or automatically fixing them
+for you. Linters are especially useful when collaborating with someone
+who is less experienced with coding or who contribute occasionally to
+assist in the code being consistently formatted across the project.
+
+To ensure consistent styling without needing to run checks manually,
+*automatic* linting/styling can be a great solution. That's where the
+`{styler}` package comes in!
 
```

Comment:
Should there be an actual task within a reading? I.e., adding a package to `DESCRIPTION`?

Reply:
Good point. I'll make an issue to convert it to an exercise